aws-lambda-java-runtime-interface-client/src/main/java/com/amazonaws/services/lambda/runtime/api/client/runtimeapi/NativeClient.java

```diff
@@ -44,10 +48,25 @@
             initializeClient(userAgent.getBytes());
     }
 
+    /**
+     * @return a string describing the detected architecture the RIC is executing on
+     * @throws RuntimeException
+     */
+    static String getArchIdentifier() {
+        String arch = System.getProperty("os.arch");
+
+        if (arch.matches(supported_x86_architectures)) {
+            return "x86";
+        } else if (arch.matches(supported_arm_architectures)) {
+            return "arm";
+        }
+
+        throw new UnknownPlatformException("architecture not supported: " + arch);
+    }
+
```

Comment:
Could you please use the naming/filter patterns suggested in https://github.com/carlzogh/aws-lambda-java-libs/commit/0b16ecde364fe03ea5036a9301581a1cfb9b9797? I'd also rather we don't use `x86` or `arm` but `x86_64` and `arm64` all around.

Reply:
incorporated in the last update